### PR TITLE
Filter dissociated tokens when querying tokens with account ID

### LINKED ISSUE #1594: Token REST API by account ID returns dissociated accounts
**Detailed Description**
Token REST API filtering by account ID returns both associated and dissociated accounts. As the state of being dissociated is probably not that useful for clients, we should additionally filter those token accounts where associated is true. If later customers want to also see dissociated, we can add a `associated=true|false` parameter with a default of true.

**Actual Behavior**

1. Go to https://testnet.mirrornode.hedera.com/api/v1/tokens?account.id=365301
2. Token `0.0.365303` appears in the output despite this account no longer associated with this token

**Expected Behavior**
JSON output should not show token `0.0.365303`.

**Environment:**
- Version: 0.27

**Additional Context**

### PR DESCRIPTION
**Pull request by xin-hedera**
**Detailed description**:

- Filter dissociated tokens of accounts
- Remove unused dependencies `express-oas-generator` and `node-cache`

**Which issue(s) this PR fixes**:
Fixes #1594 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

**Review by steven-sheehy — DISMISSED** [review]
LGTM. Just some code smells and failed tests.

File: hedera-mirror-rest/tokens.js [hunk 1]
```diff
@@ -78,23 +97,15 @@ const extractSqlFromTokenRequest = (pgSqlQuery, pgSqlParams, nextParamCount, fil
...
-    pgSqlQuery += applicableFilters === 0 ? ` where ` : ` and `;
-    applicableFilters++;
-    pgSqlQuery += `${filterColumnMap[filter.key]}${filter.operator}$${nextParamCount++}`;
-    pgSqlParams.push(filter.value);
+    conditions.push(`${filterColumnMap[filter.key]}${filter.operator}$${params.push(filter.value)}`);
   }
 
-  // add order
-  pgSqlQuery += ` order by ${sqlQueryColumns.TOKEN_ID} ${order}`;
+  const whereQuery = conditions.length !== 0 ? `where ${conditions.join(' and ')}` : '';
```
Comment (Nana-EC) [inline hedera-mirror-rest/tokens.js:103]: I like this better 👍🏾

File: hedera-mirror-rest/__tests__/tokens.test.js [hunk 2]
```diff
@@ -95,17 +97,20 @@ describe('token extractSqlFromTokenRequest tests', () => {
       },
     ];
 
-    const expectedquery =
-      'select t.token_id, symbol, e.key from token t join t_entities e on e.id = t.token_id where e.ed25519_public_key_hex = $1 order by t.token_id asc limit $2;';
+    const expectedquery = `select t.token_id, symbol, e.key from token t
+        join t_entities e on e.id = t.token_id
+        where e.ed25519_public_key_hex = $1
+        order by t.token_id asc
+        limit $2`;
```
Comment (xin-hedera) [inline hedera-mirror-rest/__tests__/tokens.test.js:104]: code smell says this line exceeded 256 characters

Comment (xin-hedera) [inline hedera-mirror-rest/__tests__/tokens.test.js:104]: code smell says this line exceeded 256 characters

**Review by Nana-EC — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Review by ijungmann — APPROVED** [review]

**Pull request merged**